src/Sidekick.Infrastructure/PoeApi/Items/Modifiers/AlternateModifierProvider.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Sidekick.Infrastructure.PoeApi.Items.Modifiers.Models;
+
+namespace Sidekick.Infrastructure.PoeApi.Items.Modifiers
```

Comment:
As the source for this data is Repoe, I think the namespace for alternate modifiers should be something like : 
Sidekick.Infrastructure.Repoe.Modifiers

Reply:
Also update the file structure accordingly :)